backend/migrations/20230703130438_food.up.sql

```diff
@@ -1,8 +1,8 @@
 -- Add up migration script here
-CREATE TYPE meal_type AS ENUM ('VEGAN','VEGETARIAN','BEEF','BEEF_AW','PORK','PORK_AW','FISH','UNKNOWN');
```

Comment:
You are not allowed to change old database migrations. Otherwise, they cannot be applied to the production database.
If you want to change something, you would need to create a new migration and use SQL functionality to rename tables etc. However, this is a hassle most of the time. Why do you want to rename this?

Reply:
To make it consistent with the frontend and to generalise the concept of a meal type, because sides also have a type

Reply:
You are right, that would be the better name

Reply:
I changed it back in the api. Everywhere else is now adjusted. For the future, we could update the app to use both and when 95% (or more) of people have the new app, then we can change the API. That way everything is consistent between front and backend and there are very few people, who notice a change